text/0000-support-insecure-registries.md

```diff
@@ -0,0 +1,74 @@
+# Meta
+[meta]: #meta
+- Name: Support for pull images from or push images to insecure image registries.
+- Start Date: 2022-08-17
+- Author(s): wanjunlei
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This RFC describes how to pull images from or push images to insecure image registries when building images using buildpacks.
+
+# Definitions
+[definitions]: #definitions
+
+Insecure image registry - The insecure image registry mentioned in this RFC refers to the image registry using the http protocol, and with a non-internal ip or a domain name. Because currently buildpacks can access insecure image registries using internal ip.
+
+# Motivation
+[motivation]: #motivation
+
+To fix issue [Support insecure registries in non-daemon case](https://github.com/buildpacks/lifecycle/issues/524).
+
+To pull images from insecure image registries when building images using buildpacks, and push target images to insecure image registries after the build is complete.
+
+# What it is
+[what-it-is]: #what-it-is
+
+With this RFC, user can use images in insecure image registries to build image, and push the compiled image to the insecure image registries like this.
+
+```shell
+creator --run-image=develoment-registry.com/run-java:v16 --insecure-registry=develoment-registry.com --insecure-registry=testing-registry.com testing-registry.com/java-sample:latest
+```
+
+The flag `--insecure-registry` will add to analyzer, export, restor, rebaser and image tool too.
```

Comment:
I apologize for the lack of context. What's "image tool"?

Reply:
The `image tool` is [this](https://github.com/buildpacks/lifecycle/blob/main/tools/image/main.go#L56).